lib/src/main/java/io/ably/lib/http/AnySyncHttp.java

```diff
@@ -0,0 +1,65 @@
+package io.ably.lib.http;
+
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.Callback;
+import io.ably.lib.types.ErrorInfo;
+
+/**
+ * Created by tcard on 28/04/2017.
+ */
+
+public class AnySyncHttp {
```

Comment:
Should we perhaps add a brief comment as to the purpose of this class for future us?